postgres/src/main/java/org/jdbi/v3/postgres/internal/BitStringEnumSetColumnMapper.java

```diff
@@ -44,8 +45,18 @@
         }
 
         return IntStream.range(0, bits.length())
+            .peek(checkIfBitIn(bits))
             .filter(i -> bits.charAt(i) == '1')
             .mapToObj(i -> enumConstants[i])
             .collect(toCollection(() -> EnumSet.noneOf(enumType)));
     }
+
+    private static IntConsumer checkIfBitIn(String bits) {
+        return i -> {
+            char bit = bits.charAt(i);
```

Comment:
You could just fold this check into the `map`, then you save a stream pipeline and one call to `charAt`.  Not that it matters all that much...